_schema.graphql

```diff
@@ -3467,16 +3467,12 @@ type CommerceBuyOrder implements CommerceOrder {
   # Item total in cents, for Offer Orders this field reflects current offer
   itemsTotalCents: Int
   lastApprovedAt(
-    # This arg is deprecated, use timezone instead
-    convert_to_utc: Boolean
```

Comment:
Hmm... I didn't expect these to be removed from v1.

Reply:
As discussed with @ashkan18 [here](https://artsy.slack.com/archives/CA8SANW3W/p1564580357361600?thread_ts=1564573228.358600&cid=CA8SANW3W). The alternative would have required duplicating the stitching setup now, which was less preferable.